### DISCUSSION TITLE
[TOOL] Truice 2.1.6.x (Win32) AzerothCore DB support

### DISCUSSION THREAD

**Discussion by chaosua** [azerothcore/azerothcore-wotlk#18241] [Tools]
Hello. Maybe someone need this tool (only Windows OS)
https://github.com/chaosua/Truice/tree/azeroth335

I adapted base functionality of Faq/Truice 2.1.6 to work with Azeroth DB 335 with current state on 27.01.24

Latest releases search there https://github.com/chaosua/Truice/releases/ 
The last number after the period in the program version: even - Azeroth DB, odd - Trinity DB.
You can view/edit most of creature, gameobject, item, quest tables including their locales (depended on locale selected in config)

PS: I know about Keira3 existance. But in some cases Truice is more comfortable to use on Windows, maybe not only for me. Also there is Trinity 335 support branch.
I'm not delphi programmer, so do not expect big changes, but I can try make some bugfixes if you find them

**Comment by chaosua** [comment]
Release Truice 2.1.6.14 https://github.com/chaosua/Truice/releases/tag/2.1.6.14
quest_template: adapted to commit https://github.com/azerothcore/azerothcore-wotlk/commit/b6c429c5a8c3c127d9a14eef72e5bfb1a796d6bf (droped RewardBonusMoney)
smart_scripts: updated definition of actions, events, targets, added new scripts
conditions: fix columns loading, add new condition types
added support to edit page_text_locale

**Comment by virgo77** [comment]
Hi,
I'm going to try it.

One thing: is it possible to make the login screen resizable? Or maybe the DB/Table fields resizable?
As I got long Tables Names, I can't see the entire name to select the good ones :)

I'll give some reviews later. Thanks for this tool

V.

**Reply by chaosua** [reply]
i think I found that config, will be in x.16 version
![truice](https://github.com/azerothcore/azerothcore-wotlk/assets/544218/511b1ae6-b91d-42fe-8bad-1c48717dfcb7)

pre-release you can try with changes made till now 
[Truice.zip (7zip)](https://github.com/azerothcore/azerothcore-wotlk/files/14318375/Truice.zip)

**Reply by virgo77** [reply]
The ZIP file does not work on my computer. I got this error:
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/23108339/85df8f51-3033-4db2-89bd-8391ecd512ec)

Seems not to be blocked by Windows Defender, as I don't have alert.
Maybe a too hard compression level?

Don't worry, I can wait the next build :)

Thanks a lot

V.

**Reply by chaosua** [reply]
i think it because not standart compresion method I used in 7zip for zip format

**Reply by virgo77** [reply]
pfff, i'm sorry about that. As it was a zip i didn't tried 7zip. So it works, obviously :)
Thanks. :)

**Comment by chaosua** [comment]
Release Truice 2.1.6.16 https://github.com/chaosua/Truice/releases/tag/2.1.6.16
added support to edit broadcast_text, broadcast_text_locale tables
some cosmetic and logic changes

**Comment by chaosua** [comment]
Release Truice 2.1.6.20 https://github.com/chaosua/Truice/releases/tag/2.1.6.20

**Comment by WillyFennec** [comment]
Hello! 😊
Just thank you for yor work.
Keira is good, but Truice is so habitual.

**Comment by Danmi100** [comment]
I have been using truice for a very long time, and this news somehow passed me by that you decided to adapt it to ACDB... Thank you for your work!!!

**Comment by chaosua** [comment]
Release Truice 2.1.6.22 https://github.com/chaosua/Truice/releases/tag/2.1.6.22

**Comment by chaosua** [comment]
Release Truice 2.1.6.24 https://github.com/chaosua/Truice/releases/tag/2.1.6.24
